project_administrator_restricted_visibility/__manifest__.py

```diff
@@ -0,0 +1,13 @@
+{
+    "name": "Project Administrator Restricted Visibility",
+    "version": "16.0.1.0.0",
+    "summary": "Adds a 'Project Administrator' access group "
+    "with restricted visibility to 'Projects'",
+    "author": "Tecnativa, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/project",
+    "license": "AGPL-3",
+    "category": "Project",
```

Comment:
Please add yourself and @rafaelbn as [maintainers](https://doc.moduon.team/doc/estandares-de-contribucion-gxxTTBnNnb#h-mantenedores).

Reply:
Done!